test/cypress/dashboards/integration/Components/general.cy.js

```diff
@@ -38,3 +38,42 @@ describe('Caption', () => {
       .should('be.visible');
   });
 });
+
+describe('Data polling restarting', () => {
+  before(() => {
+    cy.visit('dashboards/data/connector-polling');
```

Comment:
if this demo is only for the sake of this test, please move it to `dashboards/cypress/...`

Reply:
Changed, thanks!